### DISCUSSION TITLE
Tests corrección T2

### DISCUSSION THREAD

**Discussion by Segismundios** [IIC2133-PUC/2024-1#188] [Tarea 2] [answered]
Hola, me preguntaba si los tests con que se corrigió la T2 son los mismos que tuvimos para desarrollar la tarea. Me interesa saberlo porque localmente paso todos los tests, pero en la correción me aparece error en algunos. Es particularmente extraño que arrojó error en tests easy, mientras que pasó los hard. Creo que esta información sería útil en los dos casos:
1)En caso de que sean los mismos tests, estoy seguro de mandar a recorregir, porque aparentemente habría un error al momento de la corección.
2)En caso de que sean tests distintos, creo que podría ser útil que eventualmente se liberen esos tests, para poder identificar los errores que tuve en el código, y así también se evita una solucitud de recorrección que no corresponde.

**Comment by benjavicente** [comment, marked as answer]
Hola! Los test son referenciales, distintos a los que usamos para evaluar, y se los entregamos para que puedan tener una idea de la magnitud de los test que se usarán para evaluar, y puedan más fácilmente encontrar errores y oportunidades de mejora.

Por lo que revise, te falla el test case que sale en el enunciado:

```txt
8
ATCGGATC
3
2
ATC
GCT
```

```txt
Consulta 0
0
5
Consulta 1
```

Voy a añadir los test que se usaron para evaluar, pero por lo que he visto, creo que si falla en ese test, falla en esos easy. 

Te recomiendo que mandes a recorregir con cambios de código. Si son cambios menores, puedes arreglar el código y que funcione en esos casos sin tener un descuento asociado.

**Comment by Segismundios** [comment]
Valee, gracias!
